smarts/env/gymnasium/hiway_env_v1.py

```diff
@@ -308,7 +328,7 @@ def step(
                 info,
             )
```

Comment:
Assume user configures `HiWayEnvV1` with inputs `environment_return_mode=EnvReturnMode.per_agent` and `observation_options=ObservationOptions.full`. What would be the step `rewards`, `terminated`, and  `truncated` returned for inactive agents?